dotcom-rendering/docs/snyk/how-to.md

```diff
@@ -0,0 +1,21 @@
+## How to run Snyk locally?
+
+1. Install the Snyk CLI
+   * using npm: `npm install snyk -g`
+   * using yarn: `yarn global add snyk`
+2. Run `snyk auth`
+3. Run:
+   * For TS / JS repos: `snyk test --severity-threshold=high --file=package.json` or `snyk test --severity-threshold=high --file=package-lock.json`.
+   * For Scala repos: `snyk test --severity-threshold=high --file=build.sbt`
+
+This is especially helpful if you want to know early in the development cycle whether you have fixed a vulnerability or to make sure you haven't introduced a new one.
+
+
+## How to ignore a vulnerability?
+
+There are two ways of ignoring a vulnerability:
+1. By running `snyk ignore`. See more details here on how to run [this CLI command](https://docs.snyk.io/snyk-cli/commands/ignore#id-less-than-issue_id-greater-than). Running this will generate a `.snyk` file if there isn't one, or it will update the existing one. The big advantage of this option is `.snyk` can be version controlled.
+2. By using the UI you can get the following options when you click "Ignore":
```

Comment:
Maybe worth adding a "not recommended" or something to this method, to highlight the value in having a version controlled ignore file?!

Reply:
Is there a benefit to leaving this off if it's not recommended?

Reply:
This might be something we would want to discuss further with DevX but for now I have kept only the UI option. See here: https://github.com/guardian/frontend-lambda/pull/28#pullrequestreview-1298153649